app/screens/VenueScreen/VenueScreen.tsx

```diff
@@ -99,14 +107,21 @@ export const VenueScreen: FC<TabScreenProps<"Venue">> = () => {
   useHeader({ title: translate("venueScreen.title") })
 
   return (
-    <Screen style={$root} preset="fixed" contentContainerStyle={$container}>
-      <SectionList
-        stickySectionHeadersEnabled
-        sections={SECTIONS}
-        renderSectionHeader={({ section: { title } }) => (
-          <Text preset="screenHeading" style={$heading}>
-            {title}
-          </Text>
+    <Screen style={$root} preset="fixed">
+      <FlatList
+        ListHeaderComponent={
+          <>
+            <Workshops />
+            <Text preset="screenHeading" style={$heading}>
+              {translate("venueScreen.thanksToThisYearsSponsors")}
+            </Text>
+          </>
+        }
```

Comment:
hmm... I see what you did here, but it's technically not a header to the render items 🤔 (TOL I think this could cause some accessibility because semantically ListHeaderComponent should be a header to renderItem, right?)

Reply:
Good point, I hadn't considered that. It was mostly to get rid of the nested virtualized scrollview warning......will rethink

Reply:
Oooh I think I know that warning... would love to pair if you're game to!

Reply:
fixed, thanks for the pairing!